service/hook/common/metrics.go

```diff
@@ -0,0 +1,220 @@
+package common
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+// Event ...
+type Event string
+
+const (
+	// PushEvent ...
+	PushEvent Event = "git_push"
+	// PullRequestEvent ...
+	PullRequestEvent Event = "pull_request"
+)
+
+// Action ...
+type Action string
+
+const (
+	// PushCreatedAction ...
+	PushCreatedAction Action = "created"
```

Comment:
When you push the first commit on a branch, a push created event is emitted, because other than pushing a commit a new branch was created, other push events are the git_push:pushed metrics.

Reply:
I see, so created is actually branch created? Could we rename it to reflect that, and make it easier for any future readers of the code to understand this difference?

Reply:
The data pipeline builds on the current actions, so I wouldn't change the action name for now. Instead, I added comments explaining what event each action represents in https://github.com/bitrise-io/bitrise-webhooks/pull/139/commits/c4e05af4a11bd098fc542e57dcbc3ad830f631a6